src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/NoticeOfChangeController.java

```diff
@@ -75,12 +76,13 @@ public GenericCallbackResponse nocSubmitted(@RequestBody CallbackRequest callbac
         GenericCallbackResponse callbackResponse = new GenericCallbackResponse();
 
         if (APPLY_NOC_DECISION.equals(callbackRequest.getEventId())) {
-            CaseData caseData = callbackRequest.getCaseDetails().getCaseData();
+            CaseDetails caseDetails = callbackRequest.getCaseDetails();
+            CaseData caseData = caseDetails.getCaseData();
 
             //send emails here
             try {
                 nocNotificationService.sendNotificationOfChangeEmails(
-                        callbackRequest.getCaseDetailsBefore().getCaseData(), caseData,
+                        callbackRequest.getCaseDetailsBefore(), caseDetails,
```

Comment:
should each param be on its own line?

Reply:
Nah I don't think it makes much difference